custom-template/package.json

```diff
@@ -0,0 +1,13 @@
+{
```

Comment:
I have added the custom template so I can inline svg definitions into head of index. I will try to inline svg definitions somehow different, maybe to have section icon definitions that has markup of svg icons. Will look into it.